app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False) \
```

Comment:
Place each argument in the `__init__` on the new line including `self` so there will be no arguments on the same line with opening and closing parenthesis. It will help to improve the readability and avoid using line breaking.

Reply:
Done